### DISCUSSION TITLE
Welcome to Human-GEM Discussions!

### DISCUSSION THREAD

**Discussion by JonathanRob** [SysBioChalmers/Human-GEM#219] [General]
## Welcome!
We are using GitHub Discussions as a place to connect with the community. Here you can:
* Ask questions about the Human-GEM repository or model
* Share ideas
* Provide feedback/suggestions

**Comment by cmaldaz** [comment]
Hi
A very naive question
I was exploring your very interesting database
In what exactly is the ‘gene rule’ of any metabolic specific reaction based on? 
Just literature searches or other experimental based evidence or database?
Please let me know to understand better your resource
Thank you

**Reply by JonathanRob** [reply]
Hi @cmaldaz, the gene rules are primarily based on literature, which may include experimental or other indirect evidence that the gene(s) facilitate the reaction. These references (PubMed IDs) are often (but not always) associated with the reaction, so one can look into the literature upon which the corresponding rule is based.

In Human-GEM (and many other GEMs), keep in mind that many of these rules have been carried over from earlier model or database iterations, and therefore may contain errors or missing information. The curation of the gene rules is therefore an ongoing process which benefits greatly from users reporting errors as they are found.
